lib/wda/webdriveragent.js

```diff
@@ -119,7 +122,9 @@ class WebDriverAgent {
       if (this.prebuildWDA) {
         await this.xcodebuild.prebuild();
       }
-      return await this.xcodebuild.start();
+      return await retryInterval(WDA_LAUNCH_RETRY, WDA_LAUNCH_DELAY, async () => {
+        return await this.xcodebuild.start();
```

Comment:
This can be rewritten as
```js
return await retryInterval(WDA_LAUNCH_RETRY, WDA_LAUNCH_DELAY,
  this.xcodebuild.start.bind(this.xcodebuild));
```

But I agree that this just repeats the problem if there _is_ a problem.